cfy_manager/components/restservice/restservice.py

```diff
@@ -193,20 +196,24 @@ def _verify_restservice(self):
     def _verify_restservice_alive(self):
         systemd.verify_alive(RESTSERVICE)
 
-        logger.info('Verifying Rest service is working as expected...')
-        self._verify_restservice()
+        if not config[CLUSTER][JOIN_CLUSTER]:
+            logger.info('Verifying Rest service is working as expected...')
+            self._verify_restservice()
 
     def _configure_db(self):
         configs = {
             'rest_config': REST_CONFIG_PATH,
             'authorization_config': REST_AUTHORIZATION_CONFIG_PATH,
             'security_config': REST_SECURITY_CONFIG_PATH
         }
-        if config[CLEAN_DB]:
+        if config[CLEAN_DB] and not config[CLUSTER][JOIN_CLUSTER]:
```

Comment:
I think it would be cleaner to just do:
```
if config[CLUSTER][JOIN_CLUSTER]:
    self.logger.info('Joining cluster during bootstrap, ignoring DB '
                              'configuration')
    return
```

Reply:
Agree, fixed